src/test/kotlin/net/lumalyte/lg/infrastructure/persistence/migrations/BannermanMigrationTest.kt

```
@@ -0,0 +1,213 @@
+        return try {
+            connection.createStatement().use { stmt ->
+                stmt.executeQuery("PRAGMA table_info($tableName)").use { rs ->
+                    while (rs.next()) {
+                        if (rs.getString("name") == columnName) {
+                            return true
+                        }
+                    }
+                    false
+                }
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)